src/pages/comparison/interfaces/filter.interface.ts

```diff
@@ -0,0 +1,27 @@
+export interface InpSwitch {
+  name?: string;
+  id?: string;
+  value?: string;
+  lable?: string;
+  posTime?: number;
+  time?: string;
+  delta?: number;
+  min?: string;
+  max?: string;
+  step?: string;
+  refAttribute: React.Ref<HTMLInputElement>;
+  units?: string;
+}
+
+export interface Body {
```

Comment:
Consider to use [`Partial`](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialt) instead of manually set optional fields